src/main/java/cn/nukkit/block/BlockBed.java

```diff
@@ -25,6 +26,7 @@
  * author: MagicDroidX
  * Nukkit Project
  */
+@Log4j2
```

Comment:
This should be removed

Reply:
I hate it when I forget to remove the annotation after removing all the log calls